package.json

```diff
@@ -1,115 +1,116 @@
 {
```

Comment:
I can't figure out why git thinks this while file has changed.

all that is different is is installed the library `usehooks-ts`. The lock file still looks correct